.github/workflows/main.yml

```diff
@@ -69,11 +76,16 @@ jobs:
       with:
         name: docs
         path: Docs/_site/
-    - run: Docs/publish.sh
+    - if: >-
+        github.event_name != 'pull_request' &&
+        github.repository == 'planetarium/libplanet'
+      run: Docs/publish.sh
       env:
         GHPAGES_SSH_KEY: ${{ secrets.GHPAGES_SSH_KEY }}
-      if: github.event_name != 'pull_request'
-    - uses: shawnbot/action-status@master
+    - if: >-
+        github.event_name != 'pull_request' &&
+        github.repository == 'planetarium/libplanet'
+      uses: shawnbot/action-status@master
```

Comment:
I believe these two changes are unnecessary, because it's enough that the `docs` job in itself is under the same condition.  These changes also affect other active contributors' workflow as well because we sometimes check how things we've added look like in the docs, before we submit a pull request to the upstream.

Reply:
I found that the [Run shawnbot/action-status@master](https://github.com/gurrpi/libplanet/runs/511138510?check_suite_focus=true) step failed on the forked repository when triggered by `on: push`.
```
cat: Docs/obj/url.txt: No such file or directory
8
{ HttpError: Bad credentials
9
    at response.text.then.message (/action-status/node_modules/@octokit/request/lib/request.js:55:27)
10
    at process._tickCallback (internal/process/next_tick.js:68:7)
11
  name: 'HttpError',
12
  status: 401,

<below is intentionally omitted. See above link to see full message.>
```

Reply:
I understand that the ` Docs/publish.sh` step should not have condition.